bzira.py

```diff
@@ -265,7 +265,7 @@ def proces(bug, bugs):
         
     issue_dict = create_proxy_jira_dict(options, bug)
 
-    if(issue_dict):
```

Comment:
nah, it can fail further down too. I've added code that handles it by catching exceptions.

could be cleaned up some more, but its a start.